SYCL/CMakeLists.txt

```diff
@@ -19,8 +19,12 @@ if(CHECK_SYCL_ALL)
 
     string(REPLACE "," "_" TARGET check-sycl-${TARGET_BE}-${TARGET_DEVICES})
 
+    set(TEST_DIR ".")
+    if(${TARGET_BE} MATCHES "esimd_emulator")
+      set(TEST_DIR "./ESIMD/")
+    endif()
     add_custom_target(${TARGET}
-      COMMAND python3 ${TEST_SUITE_LIT} ${TEST_SUITE_LIT_FLAGS} --param sycl_be=${TARGET_BE} --param target_devices=${TARGET_DEVICES} .
+      COMMAND python3 ${TEST_SUITE_LIT} ${TEST_SUITE_LIT_FLAGS} --param sycl_be=${TARGET_BE} --param target_devices=${TARGET_DEVICES} ${TEST_DIR}
```

Comment:
Can we sink this logic to lit.cfg and adjust `config.test_source_root` (and probably `config.test_exec_root`)?